CHANGELOG.md

```diff
@@ -1,12 +1,97 @@
+
+#### Funding page
+
+oranda now has the ability to autodetect whether you're using GitHub's funding embed functionality (meaning you have a `.github/FUNDING.yml`), in which case it'll automatically generate a page showing your available funding options.
+
+Additionally, you can enhance this page by selecting a particular funding channel to be showcased as well as providing custom content from a `funding.md` Markdown file. You can, of course, fully disable this feature, or adjust the paths if you want.
+
+This is less of a defined feature and more of an experiment on how we can better integrate maintainers' funding sources onto their websites. Please let us know what you think, or if you have any other feedback or input!
+
+#### `mdbook` autodetect and styling
+
+oranda themes now get applied to your mdbook output, too, meaning there's less of a discrepancy between your flashy oranda page and your default-mdbook-styled docs. We've also been hard at work being able to detect when you use `mdbook` without you having to set it, which should now work in the majority of cases.
+
+### Configuration
+
+#### New configuration structure
+
+We've completely revamped our configuration structure to be more future-proof and to require less refactoring in the fullness of time. For the new configuration layout, please [consult the docs](https://opensource.axo.dev/oranda/book/configuration.html).
+
+One other major change is that we now **reject unknown config keys**. This means that if you've had a oranda 0.0.3 site, it will now force you to migrate your config to the new format. We've decided on this because we believe that doing anything but hard erroring in this situation would lead to unwanted behavior (old keys getting interpreted in new, weird ways, and so on).
+
+#### Config schema
+
+oranda's configuration schema is now available in a JSON schema for each release starting with this one. This means that in editors like VSCode, you can attach the schema to your `oranda.json` file and get autofill and documentation, like this:
+
+```json
+{
+  "$schema": "https://github.com/axodotdev/oranda/releases/download/v0.1.0/oranda-config-schema.json"
+}
+```
+
+### CLI
+
+#### `dev` command
+
+This release introduces `oranda dev`, which bundles both building your site and serving it from a file server, as well as sprinkling in auto-recompilation when files change. It's intended to be the prime command for local development of a oranda site.
+
+## Bug fixes
+
+(this is a selection, there's been way too many to fully list, at least until we're able to automatically generate a list)
+
+- Various style fixes, lists now display correct, colors should be less offensive to the eye, that sort of stuff
+- We're much better now at handling complex release histories!
+- We've completely removed the `version` key from the configuration. It wasn't used, and we probably won't use it in the future, either
```

Comment:
completely forgot it but we should totally gloat over oranda now being multitudes faster than in 0.0.3